official/resnet/keras/resnet_model.py

```diff
@@ -204,12 +218,13 @@ def resnet50(num_classes, dtype='float32', batch_size=None):
                     kernel_regularizer=regularizers.l2(L2_WEIGHT_DECAY),
                     name='conv1')(x)
   x = layers.BatchNormalization(axis=bn_axis,
+                                scale=False,
                                 momentum=BATCH_NORM_DECAY,
                                 epsilon=BATCH_NORM_EPSILON,
+                                fused=True,
                                 name='bn_conv1')(x)
   x = layers.Activation('relu')(x)
-  x = layers.ZeroPadding2D(padding=(1, 1), name='pool1_pad')(x)
```

Comment:
Is there any quality drop after removing the zero padding?

Reply:
I am kicking off a run now and will link the kokoro job so we can wait and see.  easy to do.

Reply:
Great thanks! Note that zero padding is not removed, it's just fused into the followup max pooling layer.

Reply:
Thanks for checking the accuracy! It's indeed hurting the accuracy, I will look into this.

For now I reverted the zero padding change. Finished a convergence test and got 76.09% which is probably acceptable. We will closely watch the accuracy numbers these days.